pkg/controllers/register.go

```diff
@@ -120,6 +121,12 @@ func RegisterControllers(ctx *context.ControllerContext, syncers []syncer.Object
 		return err
 	}
 
+	// register controller that maintains pod security standard check
+	err = registerPodSecurityController(ctx)
```

Comment:
only register controller if pod security standard is enabled

Reply:
pod security standard will be enabled by default with privileged.